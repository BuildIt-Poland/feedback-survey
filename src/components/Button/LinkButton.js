import React from 'react';
import { Link } from 'react-router-dom';
import SurveyButton from '.';

const LinkButton = ({ linkTo = '/', label }) => {
  return (
    <Link to={linkTo}>
      <SurveyButton label={label} />
    </Link>
  );
};

export default LinkButton;
