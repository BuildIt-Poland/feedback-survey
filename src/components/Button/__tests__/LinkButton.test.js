import React from 'react';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import LinkButton from '../LinkButton';

describe('COMPONENT - SurveyButton LinkButton', () => {
  it('renders correctly', () => {
    const props = {
      linkTo: '/',
      label: 'test'
    };

    const component = create(
      <MemoryRouter>
        <LinkButton {...props} />
      </MemoryRouter>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
