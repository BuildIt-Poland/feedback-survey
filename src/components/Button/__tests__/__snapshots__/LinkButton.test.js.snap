// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - SurveyButton LinkButton renders correctly 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <button
    className="sc-bdVaJa hZuWlY StyledButton-sc-323bzc-0 bmExQD"
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    size="large"
    type="button"
  >
    test
  </button>
</a>
`;
